src/test/java/nl/tudelft/jpacman/board/BoardFactoryTest.java

```diff
@@ -18,79 +20,75 @@
 	 * The factory under test.
 	 */
 	private BoardFactory factory;
-	
+
+	/**
+	 * Squares on the board to test.
+	 */
+	private Square s1, s2;
+
 	/**
 	 * Resets the factory under test.
 	 */
-	@Before
+	@BeforeEach
 	public void setUp() {
 		PacManSprites sprites = mock(PacManSprites.class);
 		factory = new BoardFactory(sprites);
+
+		s1 = new BasicSquare();
+		s2 = new BasicSquare();
 	}
 	
 	/**
 	 * Verifies that a single cell is connected to itself on all sides.
 	 */
 	@Test
 	public void worldIsRound() {
-		Square s = new BasicSquare();
-		Square[][] grid = new Square[][]{{s}};
-		factory.createBoard(grid);
-		assertEquals(s, s.getSquareAt(Direction.NORTH));
-		assertEquals(s, s.getSquareAt(Direction.SOUTH));
-		assertEquals(s, s.getSquareAt(Direction.WEST));
-		assertEquals(s, s.getSquareAt(Direction.EAST));
+		// Create a round world with just one cell.
+		factory.createBoard(new Square[][]{{s1}});
+
+		// All directions from that single cell point back to that cell.
+		Arrays.stream(Direction.values()).forEach((dir) ->
```

Comment:
Could also be:

assertThat(Arrays.stream(Direction.values()).map(s1::getSquareAt)).containsOnly(s1);